operator/controllers/syncer/csi_node.go

```diff
@@ -169,6 +169,11 @@ func (s *csiNodeSyncer) SyncCSIDaemonsetFn(daemonSetRestartedKey string, daemonS
 		Type:          strategyType,
 	}
 	out.Spec.UpdateStrategy = strategy
+	if hostNetwork == "ENABLED"{
```

Comment:
This additional check is not required here as at this point we get validated/default value

Reply:
This is not for any validation. This is just to update pod spec to true/false based on the env ENABLED/DISABLED.